crawl-ref/source/describe.cc

```diff
@@ -4501,22 +4501,26 @@ static void _describe_mons_to_hit(const monster_info& mi, ostringstream &result)
 }
 
 /**
- * Print a bar of +s and .s representing a given stat to a provided stream.
+ * Print a bar of +s representing a given stat to a provided stream.
  *
- * @param value[in]         The current value represented by the bar.
- * @param scale[in]         The value that each + and . represents.
+ * @param value[in]         The current value represented by the bar. Clamped
+ *                          to zero.
+ * @param scale[in]         The value that each + represents.
  * @param name              The name of the bar.
  * @param result[in,out]    The stringstream to append to.
- * @param base_value[in]    The 'base' value represented by the bar. If
- *                          INT_MAX, is ignored.
+ * @param base_value[in]    The 'base' value represented by the bar, clamped to
+ *                          zero. If INT_MAX, is ignored.
  */
 static void _print_bar(int value, int scale, string name,
```

Comment:
While you are at it, you could make the `string name` parameter a const reference.

Reply:
Good call, thanks!